### PR TITLE
header standardization

### PR DESCRIPTION
**Pull request by sebastien-violante**

### CODE REVIEW

File: public/assets/css/home.css [hunk 1]
```diff
@@ -1,9 +1,3 @@
-nav {
-    padding: 1em 1em 0em 1em;
-    display: flex;
-    justify-content: space-between ;
-}
-
 .questionmark-button, .music-button {
```
Comment (elodiedaubie) [inline public/assets/css/home.css]: Hi, .questionmark-button and .music-button doesn't exist anymore in the view. You should consider to delete it

File: public/assets/css/tutorialAndTOS.css [hunk 2]
```diff
@@ -48,41 +48,16 @@ hr{
...
-   
 }
+
 #logo{
     height: 10vh;
-    margin-top: 4vh;
+    margin-top: 2vh;
     margin-left: auto;
     margin-right: auto;
 }
```
Comment (elodiedaubie) [inline public/assets/css/tutorialAndTOS.css:63]: You may rename the file without uppercases

**Review by elodiedaubie — CHANGES_REQUESTED** [review]

File: public/assets/css/tutorialAndTOS.css (hunk 2 above)
Comment (sebastien-violante) [reply public/assets/css/tutorialAndTOS.css:63]: changes done

File: public/assets/css/home.css (hunk 1 above)
Comment (sebastien-violante) [reply public/assets/css/home.css]: changes done

**Review by elodiedaubie — APPROVED** [review]

**Review by RomainClair — APPROVED** [review]

**Pull request merged**